contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -737,38 +718,11 @@ handle_where_clause_restargets_left(ParseState *pstate, Node *w_clause, List *ex
 
 					if (nodeTag(xpr->lexpr) != T_ColumnRef)
 					{
-						xpr = (A_Expr *) arg;
-
-						if (nodeTag(xpr->lexpr) != T_ColumnRef)
-						{
-							if (is_sp_describe_undeclared_parameters)
-							{
-								is_supported_case_sp_describe_undeclared_parameters = false;
-								return extra_restargets;
-							}
-						}
-						ref = (ColumnRef *) xpr->lexpr;
-						field = linitial(ref->fields);
-						name = field->val.str;
-						attrno = attnameAttNum(pstate->p_target_relation, name, false);
-						if (attrno == InvalidAttrNumber)
+						if (is_sp_describe_undeclared_parameters)
```

Comment:
same as below, does it need to be two if statements?

Reply:
same explanation as below